about-this-guidebook/guidebook-governance.md

```diff
@@ -12,7 +12,7 @@ These teams are listed on GitHub as [subteams](https://github.com/orgs/CivicActi
 
 | **Directory in `docs/`**                                        | **Subteam (Working group / Practice Area)**                                    |
```

Comment:
We should update the folder names here in future pull requests.

Reply:
I just removed the table for now - I think CODEOWNERS is a better way to manage these associations.

Reply:
Agreed!